databuilder/__main__.py

```diff
@@ -160,6 +160,11 @@ def build_parser():
         help="Generate a JSON representation of the data needed for the public documentation of Backends and Contracts",
     )
     generate_docs.set_defaults(which="generate_docs")
+    generate_docs.add_argument(
```

Comment:
Nit: Perhaps add location where defaults to if no value for location path passed in.

Reply:
Not sure I'm parsing that sentence correctly… do you mean add docs to say "this will default to the current directory", setting a default to the flag, or?

The default is currently the current working directory databuilder is invoked from, but maybe I've not surfaced that enough?

Reply:
I think I was not being clear. I mean make it more clear that the documents will end up where the tool is invoked from if no location path is passed in. We might not need it. I just want to avoid people assuming if there is a `docs/` folder already in the location that they are in, that the newly generated (or updated docs) will end up in this folder logically. My read of the code is that this would not be the case.

Reply:
@CarolineMorton – updated and rebased, do you think that's clear enough?

Reply:
Looks good. thanks!